### ISSUE TITLE
Limitations of Native / elements to strip away from web

### ISSUE THREAD

**Issue by xaviervia** [batata-frita/lada#49]
## Layout

- [ ] `background.url` : It requires a primitive in Native (https://facebook.github.io/react-native/docs/images.html#background-image-via-nesting)
- [ ] `border.*.style`: is not possible to set the border style to one border specifically. Border style has to be set for the entire view

**Comment by deepsweet** [comment]
maybe let's drop `border.style` then? 🤔

**Comment by xaviervia** [comment]
Yep, I think we should drop all the ones in this list, but for now I’m just listing them.

**Comment by deepsweet** [comment]
am I wrong that technically `perspective` + `scale` can be replaced with `translate` and `width * N` + `height * N`?

**Comment by deepsweet** [comment]
>The `shadowColor`, `shadowOffset` and `shadowRadius` are props of the `View` component 

> `shadowOpacity` however is part of `style`

![why?](http://www.reactiongifs.com/r/lnwh.gif)

**Comment by deepsweet** [comment]
https://stackoverflow.com/questions/29323544/how-do-i-render-a-shadow 🤔

**Comment by xaviervia** [comment]
> am I wrong that technically perspective (and perspective.origin?) + scale can be replaced with translate and width * N + height * N?

Depends on what you mean by "technically". For a single element I think you are right, but the point of perspective is that it affect all children, so it allows you to create a "scene".

Tbh though, I don’t understand the point of this feature. If you really want to do some fancy projections why not use canvas.

**Comment by deepsweet** [comment]
>`isUppercase`, `isLowercase` and `isCapitalized` do not exist in Native. And this kind of makes sense.

I just realized this. HOLY.
